iOS_Learning/ViewController/LogInViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  LogInViewController.swift
+//  iOS_Learning
+//
+//  Created by Vlad_Rosca on 22.03.2023.
+//
+
+import UIKit
+
+class LogInViewController: UIViewController {
+    
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var loginButton: UIButton!
+    @IBOutlet weak var errorLabel: UILabel!
+    
+    private let loginService = UNLoginService()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Do any additional setup after loading the view.
+        setUpElements()
+    }
+    
+    @IBAction func loginButtonTapped(_ sender: UIButton) {
+        let error = validateFields()
+        
+        if error != nil {
+            showError(error!)
+        } else {
+            login()
+        }
+    }
+    
+    private func transitionToHome() {
+        
+        let homeViewController =
+        storyboard?.instantiateViewController(withIdentifier: Constants.Storyboard.homeViewController) as? HomeViewController
+        
+        view.window?.rootViewController = homeViewController
+        view.window?.makeKeyAndVisible()
+    }
+    
+    private func setUpElements() {
+        errorLabel.alpha = 0
+    }
+    
+    private func showError(_ error: String) {
+        errorLabel.text = error
+        errorLabel.alpha = 1
+    }
+    
+    private func validateFields() -> String? {
+        if emailTextField.text?.trimmingCharacters(in: .whitespacesAndNewlines) == "" ||
+            passwordTextField.text?.trimmingCharacters(in: .whitespacesAndNewlines) == "" {
+            return "Plase fill in all field"
```

Comment:
Start using [NSLocalizedString](https://developer.apple.com/documentation/foundation/nslocalizedstring) to have the texts translated.